tests/test_google_sheets_pagination.py

```diff
@@ -61,6 +61,34 @@ def test_run(self):
                 # verify that we can paginate with all fields selected
                 self.assertGreater(record_count_by_stream.get(stream, 0), self.API_LIMIT)
 
+                record_count_sync = record_count_by_stream.get(stream, 0)
+
+                if record_count_sync > self.API_LIMIT:
+                    primary_keys_list_1 = actual_pk_list[:self.API_LIMIT]
+                    primary_keys_list_2 = actual_pk_list[self.API_LIMIT:2*self.API_LIMIT]
+
+                    primary_keys_page_1 = set(primary_keys_list_1)
+                    primary_keys_page_2 = set(primary_keys_list_2)
+
+                    # Verify by primary keys that data is unique for page
+                    self.assertTrue(
+                        primary_keys_page_1.isdisjoint(primary_keys_page_2))
+
+                if stream == "sadsheet-pagination":
+                    # verify the data for the "sadsheet-pagination" stream is free of any duplicates or breaks by checking
+                    # our fake pk value ('id')
```

Comment:
Add a detailed comment about the data present in this sheet.

Reply:
Added